packages/migration/src/mods/7.0.0/createRefFromReact.ts

```diff
@@ -0,0 +1,40 @@
+import { mod } from 'riceburn';
+import ts from 'typescript';
+import { migration, IMigrationOptions, ModResult } from '../../migration';
+
+const commaBeforeRegex = new RegExp('\s?,\s*createRef');
+const commaAfterRegex = new RegExp('createRef\s*,\s?');
+const noCommaRegex = new RegExp('createRef');
+
+export default migration(
+  'createRef should come from React.createRef, not from office-ui-fabric-react',
+  (opts: IMigrationOptions): ModResult[] => {
+    return mod('**/*.ts?(x)', opts).asTypescript((node, modder) => {
+      if (ts.isImportDeclaration(node)) {
```

Comment:
If you get a chance to revisit this later, would you please combine all these nested if statements into a single one? They're way more annoying to read when nested.